src/chronometer.js

```diff
@@ -1,33 +1,45 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime ++
+      if (typeof printTimeCallback === 'function') {
+        printTimeCallback()
+      }
+    }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60)
   }
 
```

Comment:
Nos métodos getMinutes e getSeconds, o lab pede que seja retornado um número inteiro. Existem algumas maneiras de indicar isso e precisa acrescentar no return. 
Vc pode indicar que é um Number e todo restante do cálculo dentro de parenteses ou também indicar parseInt que retorna igualmente um número inteiro. 
return Number(Math.floor(this.currentTime / 60)) ou parseInt(return Math.floor(this.currentTime / 60))